scripts/oneoff/generate_g8_master_csv.py

```diff
@@ -0,0 +1,72 @@
+"""
+Generate a CSV (to stdout) with per-lot draft statistics for each supplier
+
+Usage:
+    scripts/oneoff/generate_g8_master_csv.py <data_api_url> <data_api_token>
```

Comment:
There is a handy function here https://github.com/alphagov/digitalmarketplace-scripts/blob/master/dmscripts/env.py#L1

That allows you to only need to put the name of the stage (plus valid token) as a command line arg (`dev|preview|staging|production`) rather than needing to know the full API URL.